### ISSUE TITLE
🛠️ [TASK] : Workspace - Scrolling & Selection 

### ISSUE THREAD

**Issue by digitalheartxs** [input-output-hk/catalyst-voices#843]
### Summary

Scrolling & Selection  logic for Workspace

#### On view: 
- [ ] On left-rail navigation selected item scroll to top.
- [ ] On selection segment (content segments in the middle) activates item in left-rail, and scroll to top 
- [ ] Only active item has active edit  - Both selection left rail / select content segments in the middle update the content in context (right rail)  
 
#### On edit:
- [ ] On edit left menu navigation is disabled / not responding 
- [ ] if edit is open, user can't scroll. 
- [ ] If edit is open, disable other edits. (grey out)
- [ ] Lock left-rail selection on edit / scroll is disabled 
- [ ] Disable Space navigation on edit + Tooltip Please close edit before navigating elsewhere?

**Comment by damian-molinski** [comment]
Blocked by #1165

**Comment by damian-molinski** [comment]
Closing in favour of #842
